### PR TITLE
Cleaned up variable scope of select test helper

### PR DESCRIPTION
**Pull request by joshuabremer**

### CODE REVIEW

File: test-support/helpers/x-select.js [hunk 1]
```diff
@@ -9,25 +9,25 @@ import jQuery from 'jquery';
...
+    throw `No options found in ${selector}`;
   }
 
   $options.each(function() {
-    let $option = Ember.$(this);
+    const $option = Ember.$(this);
 
     Ember.run(() => {
-      this.selected = texts.some((text) => {
+      const selected = texts.some((text) => {
```
Comment (joshuabremer) [inline test-support/helpers/x-select.js]: This burned me in a test. For some reason, `texts.some(func)` would return false, but `this.selected` was still set to true. As I looked through the code, I saw no reason why selected was a class property.

File: test-support/helpers/x-select.js [hunk 2]
```diff
@@ -9,25 +9,25 @@ import jQuery from 'jquery';
  * @param {string} texts - text of the option you are picking
  */
 export function select(selector, ...texts) {
-  let $select = selector instanceof jQuery ? selector : Ember.$(selector);
-  let $options = $select.find(`option`);
+  const $select = selector instanceof jQuery ? selector : Ember.$(selector);
+  const $options = $select.find(`option`);
```
Comment (joshuabremer) [inline test-support/helpers/x-select.js]: The real fix is below, but I looked through the rest of the helper and found no variables that were being set again. To make this clear I changed everything to `const`.

**Comment by joshuabremer** [comment]
Interesting....`this.selected` is there to support multiple select. Hmm....weird. Not sure why my test is failing....

**Comment by cowboyd** [comment]
Hmm, this is a good catch though, and we need to find a way to support multiselect without referencing what amounts to a global variable. Thoughts?

As an aside, we actually prefer `let` over `const`. They both indicate lexical scope, but one is both shorter and an actual English word. A preference true, but one that you'll find in effect elsewhere in the codebase.

**Pull request closed without merging**